src/flex/FlexView.js

```diff
@@ -76,6 +73,17 @@ const PropTypes = {
 @props(PropTypes, { strict: false })
 export default class FlexView extends React.Component {
 
+  componentDidMount() {
+    this.logWarnings();
+  }
+
+  logWarnings = () => {
+    const { auto } = this.props;
+    if (!t.Nil.is(auto)) {
+      warn('"auto" is DEPRECATED. Use combination of "grow", "shrink" and "basis" instead');
```

Comment:
not necessarily at once, projects should fix their flexview version to what's more appropriate (i.e. sticking to the b-r-c one, not updated, for a while if that's better) ;)